README.md

```diff
@@ -8,48 +8,59 @@ https://openslides.com for more information.
 
 __Note: OpenSlides 4 is currently under development.__
 
-### Architecture of OpenSlides 4
 
-![System architecture of OpenSlides 4](docs/OpenSlides4-systemarchitecture.png)
+## Architecture of OpenSlides 4
 
+![System architecture of OpenSlides 4](docs/OpenSlides4-systemarchitecture.png)
 
 Read more about our [concept of OpenSlides 4.0](https://github.com/OpenSlides/OpenSlides/wiki/DE%3A-Konzept-OpenSlides-4).
 
 
 ## Installation
 
-NOTE: Do not use prod at the moment. It will not work. Please refer to the DEVELOPMENT.md!
+### Requirements
 
-Required software: Docker, docker-compose, make, git
+You need [Docker](https://docs.docker.com/engine/install/) and [Docker
+Compose](https://docs.docker.com/compose/install/).
 
-For a non-development setup, clone this repo and run it via docker compose. The make command is a handy shortcut for this:
+### Setup OpenSlides
 
-    $ git clone git@github.com:OpenSlides/OpenSlides.git
-    $ cd OpenSlides
-    $ git checkout openslides4-dev  # needed, until OS4 is released
-    $ make run-prod
+Go to a nice place in your filesystem and get the [OpenSlides manage
+tool](https://github.com/OpenSlides/openslides-manage-service/releases/tag/latest).
```

Comment:
Nobody wants to go to another place. I would suggest to add a wget command: like `wget https://github.com/OpenSlides/openslides-manage-service/releases/download/latest/manage`. If you don't want to add wget as a "requirement", then you could add this line as an example:

```
Get the manage tool from .... For example with wget ...
```

Reply:
Since the user needs docker anyway, we could release the tool as docker-container. Then you would call it like this:
```
docker run manage setup --cwd
docker  run manage initial-data
...
```
Then you could add a github action to create the container. This would also work on other platforms like Windows and Mac.

Reply:
For this to work, the manage-container has to write files into the host container: https://stackoverflow.com/questions/31448821/how-to-write-data-to-host-file-system-from-docker-container

(Or it can write the files into its own container. Then all configuration is bundled in this container and we do not need the xdg path)